push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/NotificationDispatcher.java

```diff
@@ -43,12 +47,22 @@
 
     private final Logger logger = LoggerFactory.getLogger(NotificationDispatcher.class);
 
+    /**
+     * Topic to which a "success" message will be sent if a push message was successfully send and "failure" message otherwise.
+     */
+    public static final String KAFKA_PUSH_MESSAGE_SENDING_METRICS_TOPIC = "agpush_pushMessageSendingMetrics";
+    public static final String KAFKA_PUSH_MESSAGE_SENDING_SUCCESS = "agpush_success";
+    public static final String KAFKA_PUSH_MESSAGE_SENDING_FAILURE = "agpush_failure";
+    
     @Inject
     @Any
     private Instance<PushNotificationSender> senders;
 
     @Inject
     private PushMessageMetricsService pushMessageMetricsService;
+   
+    @Producer
+    private SimpleKafkaProducer<String, String> pushMessageSendingMetricsProducer;
```

Comment:
If we change the topic names we can probably change the name of the producer, too